.rubocop.yml

```diff
@@ -0,0 +1,188 @@
+AllCops:
+  TargetRubyVersion: 2.4
+  DisabledByDefault: true
+
```

Comment:
Are there any files or directories that we should exclude? I know the `db/schema.rb` is sometimes excluded, but I don't know why TBH.

Reply:
You're right, normally one would exclude e.g. the `db` and `bin` directories, just because the code there is not really regarded as "app code" and linting it often times does not make too much sense.

In our case, since we disable all cops, this is not really a problem for now. We don't have any cops that could be problematic for the schema file, migrations or the bin directory.

Reply:
Aha 💡 .  Thanks 🙇